app-games/src/main/java/com/aptoide/android/aptoidegames/editorial/EditorialViewScreen.kt

```diff
@@ -259,12 +262,10 @@ private fun ArticleViewContent(
         }
         item {
           EditorialBundle(
-            modifier = Modifier
-              .fillMaxWidth()
-              .background(color = Palette.Black),
+            modifier = Modifier.padding(top = 16.dp),
```

Comment:
Acording to figma the top padding should be 40.dp

Reply:
Fixed. It is now 16 of the bottom padding of the text + 24 of the top padding of the bundle